benchmark/sirun/appsec-iast/meta.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "appsec-iast",
+  "cachegrind": false,
+  "instructions": true,
+  "iterations": 40,
+  "variants": {
+    "no-vulnerability-control": {
+      "setup": "bash -c \"nohup node client.js >/dev/null 2>&1 &\"",
+      "run": "node --require ../../../init.js server-without-vulnerability.js",
```

Comment:
I wonder if there is a short time where the client starts requesting, but the server isn't up yet, so most of the requests will be erroring and skipped

Reply:
Server and client are automatically closed when the expected request are finished. If the server started later, it is not going to be closed automatically, and the test will fail with a timeout, because the application is not stopped. 

And if it works as you are mentioning... we have a problem in multiple benchmark tests, as appsec or http.

Reply:
true you're right, sounds good then